src/gsi/intpm2_5.f90

```diff
@@ -91,8 +92,8 @@ subroutine intpm2_5_(pm2_5head,rval,sval)
     real(r_kind) w1,w2,w3,w4,w5,w6,w7,w8
 ! real(r_kind) penalty
```

Comment:
This line is commented out in both develop and this PR.  Is there any reason to keep this commented out line?  If not, please remove it.

Reply:
The variable is not used, sure the line can be removed.